python/ribasim_testmodels/ribasim_testmodels/allocation.py

```diff
@@ -1065,3 +1071,123 @@ def allocation_example_model():
     )
 
     return model
+
+
+def main_network_with_subnetworks_model():
```

Comment:
I'll pick up this model in https://github.com/Deltares/Ribasim/pull/851 for testing.